soccer/gameplay/tactics/defense.py

```diff
@@ -61,7 +76,20 @@ def debug(self):
     def debug(self, value):
         self._debug = value
 
-    def execute_running(self):
+    def execute_defending(self):
+
+        defender1 = self.subbehavior_with_name('defender1')
+        defender2 = self.subbehavior_with_name('defender2')
+        close_defender = defender1
+        if (main.ball().pos.mag() < constants.Field.ArcRadius * 2):
+            if (defender1.robot != None and defender2.robot != None):
+                if (not defender1.go_clear and not defender2.go_clear):
+                    if close_defender.should_clear_ball(
+                            evaluation.ball.time_to_ball(
+                                close_defender.robot)):
+                        close_defender.go_clear = True
+                        self.go_clear = True
+
         self.recalculate()
 
         goalie = self.subbehavior_with_name("goalie")
```

Comment:
Now that `defending` isn't the only main state, the chunk of code here for checking the goalie should be put into `execute_running()` so that it runs during both `clearing` and `defending`.